src/python_testing/mdns_helper.py

```diff
@@ -0,0 +1,159 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+from mobly import asserts
+from zeroconf import ServiceListener, Zeroconf
+from zeroconf.asyncio import AsyncServiceInfo, AsyncZeroconf, AsyncZeroconfServiceTypes
+
+MDNS_TYPE_OPERATIONAL = "_matter._tcp.local."
+
+'''
+A service listener for the TXT record data to get populated
+'''
+
+
+class DummyServiceListener(ServiceListener):
+
+    def add_service(self, zc: Zeroconf, type: str, name: str) -> None:
+        pass
+
+    def remove_service(self, zc: Zeroconf, type: str, name: str) -> None:
+        pass
+
+    def update_service(self, zc: Zeroconf, type: str, name: str) -> None:
+        pass
+
+
+class MdnsHelper:
```

Comment:
Can you change the name of this to indicate this is for operational?

I think some of this logic will end up being common between operational and commissionable, so I think eventually you'll end up with a generic base class that exposes an operational and commissionable (and maybe commissioner) option. Anyway, for now, if you just change the name, you can extract the common bits later.